technical-reports/resolver/index.html

```diff
@@ -48,12 +48,9 @@
   <body>
     <section id="abstract">
       <p>
-        This document describes the technical specification for a resolver
-        mechanism used to manage and resolve design tokens in complex scenarios
-        involving themes, modes, brands, and other modifiers. It extends the
-        Design Tokens Format Specification by introducing a structured way to
-        combine and override token sets to produce a final, resolved set of
-        tokens for consumption by design tools and platforms.
+        This specification extends the [format](../format/) and describes a
+        method to work with alternate values for [design tokens](../), such as
+        “light mode” and “dark mode” color themes for supporting devices.
```

Comment:
Was reading the [W3C manual of style](https://www.w3.org/guide/manual-of-style/#Status) and ran across this:

> Tim Berners-Lee expressed the goal of the custom paragraph this way, "Don't be afraid of being honest about the relevant techno-political situation." In the custom paragraph, make the case for why someone should read this draft.

In light of that, I wanted to take a crack at providing a clearer, more succinct way of describing this. This is an opinionated revision! And I would be open to changing this to better follow the styleguide